packages/react-component-library/src/components/Timeline/TimelineSide.tsx

```diff
@@ -5,10 +5,10 @@ import { Button } from '../Button'
 import { getKey } from './helpers'
 import { TimelineContext } from './context'
 import { TIMELINE_ACTIONS } from './context/types'
-import { TimelineRowProps, TimelineRowsProps } from '.'
+import { TimelineRowProps } from '.'
 
 export interface TimelineSideProps extends ComponentWithClass {
-  children: React.ReactElement<TimelineRowsProps>
+  children: React.ReactElement<any>
```

Comment:
Does this this need to be `any`? Do we know what the type of `children` is?

Reply:
I'd ignore this for now the entire component is going to get reworked.